ui/components/component-library/avatar-badge/avatar-badge.stories.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+import { AvatarToken } from '../avatar-token';
+
+import Jazzicon from '../../ui/jazzicon/jazzicon.component';
+import { AvatarNetwork } from '../avatar-network';
+import { SIZES } from '../../../helpers/constants/design-system';
+import README from './README.mdx';
+import { AvatarWithBadge, badgePositions } from './avatar-badge';
+
+export default {
+  title: 'Components/ComponentLibrary/AvatarWithBadge',
+  id: __filename,
+  component: AvatarWithBadge,
+  parameters: {
+    docs: {
+      page: README,
+    },
+  },
+  argTypes: {
+    address: { control: 'text' },
+    badgePosition: { options: badgePositions, control: 'select' },
+  },
+  args: {
+    badgeProps: {
+      tokenName: 'ast',
+      tokenImageUrl: './AST.png',
+      networkName: 'Arbitrum One',
+      networkImageUrl: './images/arbitrum.svg',
+      size: SIZES.XS,
+    },
+    address: '0x5CfE73b6021E818B776b421B1c4Db2474086a7e1',
```

Comment:
Hmm interesting. I understand this could be useful to allow someone to change the badge avatar options but to avoid confusion as to which props are for which component I think we just want to expose controls for the component we are documenting. In this case, `AvatarWithBadge`. We can hard code these in the `AvatarNetwork` component.

Reply:
Updated! have removed `badgeProps` and hardcoded the props for `AvatarNetwork`